server/test/com/mirth/connect/server/api/providers/RequestedWithFilterTest.java

```diff
@@ -2,10 +2,9 @@
 
```

Comment:
I know you didn't create this file, but it should have a header as well, and you are making non-trivial changes where (I think) you can add yourself to the copyright.